CMakeLists.txt

```diff
@@ -10,6 +10,16 @@ set(CMAKE_AUTORCC ON)
 set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
+# Don't allow Qt's "slots" keyword because that mucks up Python's includes
+# NOTE:
+# +---------+-----------+
```

Comment:
I just wanna point this out because it's rather important to note for future if we need any more signals/slots